src/config/_spec.py

```diff
@@ -0,0 +1,46 @@
+"""
+_validator -   # FIXME teplates
+
+
+
+Copyright (c) 2019 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+import typing
+
+import pluggy
+
+from ._constants import _PLUGIN_NAME
+
+validator_spec = pluggy.HookspecMarker(_PLUGIN_NAME)
+rehash_spec = pluggy.HookspecMarker(_PLUGIN_NAME)
```

Comment:
[Style] `[C0103(invalid-name), ] Constant name "rehash_spec" doesn't conform to UPPER_CASE naming style`